clients/admin-ui/src/features/privacy-requests/RequestRow.tsx

```diff
@@ -256,22 +275,27 @@ const RequestRow: React.FC<{
                 size="xs"
                 mr="-px"
                 bg="white"
-                onClick={handleModalOpen}
-                _loading={{
```

Comment:
The loading styles are no longer needed because the modals have loading spinners.